cfme/infrastructure/virtual_machines.py

```diff
@@ -408,7 +433,11 @@ def delete_all(self, cancel=False):
 
         def revert_to(self, cancel=False):
             self._nav_to_snapshot_mgmt()
-            self.snapshot_tree.click_path(*self.snapshot_tree.find_path_to(re.compile(self.name)))
+            if self.name is not None:
+                self._snapshot_click_helper(self.name)
+            else:
+                self._snapshot_click_helper(self.description)
```

Comment:
consider replacing this with ``self._snapshot_click_helper(self.name or self.description)``